### PR TITLE
Update VB async sample link

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

Updated the links to the sample code @IEvangelist ported.

@Youssef1313 

Contributes to dotnet/samples#1872

### CODE REVIEW

File: docs/visual-basic/language-reference/modifiers/async.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ For more information and examples, see [Async Return Types](../../programming-gu
 
 ## Example
 
-The following examples show an async event handler, an async lambda expression, and an async method. For a full example that uses these elements, see [Walkthrough: Accessing the Web by Using Async and Await](../../programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md). You can download the walkthrough code from [Developer Code Samples](https://code.msdn.microsoft.com/Async-Sample-Accessing-the-9c10497f).
```
Comment (Youssef1313) [inline docs/visual-basic/language-reference/modifiers/async.md:57]: The code below doesn't exactly match the one from `SerialAsyncExample`.

Comment (adegeo) [reply docs/visual-basic/language-reference/modifiers/async.md:57]: I know. But these are 7 years old and would need to be rewritten. Considering we're not investing in VB docs at this time, it's not a priority. 😿

**Review by IEvangelist — APPROVED** [review]
Looks good to me, :shipit: when you're ready.

**Review by BillWagner — APPROVED** [review]
This LGTM as well. Go ahead and :shipit:

**Pull request merged**